templates/nro/.circleci/config.yml.tmpl

```diff
@@ -174,33 +210,33 @@ job_definitions:
         type: boolean
         default: false
     steps:
+      - checkout:
+          path: /home/circleci/checkout
       - attach_workspace:
           at: /tmp/workspace
       - run: activate-gcloud-account.sh
       - run: make prepare-helm
       - run: make backup
       - run: BUILD_TAG=build-$(cat /tmp/workspace/var/circle-build-num) make deploy-helm
       - run: make post-deploy
+      - run:
+          name: Extract workflow from commit message
+          command: |
+            UNHOLD_WORKFLOW_LINE=$(git --git-dir=/home/circleci/checkout/.git log --format=%B -n 1 "$CIRCLE_SHA1" | { grep '^\/unhold ' || true; } )
+            echo "line $UNHOLD_WORKFLOW_LINE"
+            if [[ -n $UNHOLD_WORKFLOW_LINE ]]; then
+              WORKFLOW_ID=${UNHOLD_WORKFLOW_LINE#"/unhold "}
+              echo "$WORKFLOW_ID"
+              echo $WORKFLOW_ID > /tmp/workspace/approve_workflow
+            fi
+      - approve_job:
+          job_name: instance-ready
```

Comment:
Shouldn't we do a check here so this only goes to test instances?

Reply:
The step is already optional, and will do nothing if no workflow ID was in the commit message, which is the case for a manual deploy of the instance.